Tests/AppTests/AnalyzerTests.swift

```diff
@@ -34,6 +37,7 @@ class AnalyzerTests: AppTestCase {
             return false
         }
         Current.fileManager.createDirectory = { path, _, _ in checkoutDir = path }
+        Current.git = .live
```

Comment:
In the first end-to-end analysis test I didn't mock out the git commands, because we're snapshotting what commands we run and it's a good test to have in place, in case we accidentally mess with the command sequence.